utils.c

```diff
@@ -108,34 +110,31 @@ int list_dir(connection_t *connection_prop, char *html, unsigned int bufsize, bo
             //Scaling the file's size
             unsigned long long int size = f_prop.st_size;
             if (size < 1024) {
-                measure="B";
+                measure=" B";
             } else if ((size = (size / 1024)) < 1024) {
-                measure="KiB";
+                measure=" KiB";
             } else if ((size = (size / 1024)) < 1024) {
-                measure="MiB";
+                measure=" MiB";
             } else {
                 size = size / 1024;
-                measure="GiB";
+                measure=" GiB";
             }
 
-            if (i % 2 == 0)
-                color = "white";
-            else
-                color = "#EAEAEA";
-
             printf_s=snprintf(html+pagesize,maxsize,
                               "<tr style=\"background-color: %s;\"><td>f</td><td><a href=\"%s\">%s</a></td><td>%lld%s</td><td>%s</td></tr>\n",
                               color, namelist[i]->d_name, namelist[i]->d_name, (long long int)size, measure,last_modified);
             maxsize-=printf_s;
             pagesize+=printf_s;
+            color = (color == "white" ? "#E0E0E0" : "white");
 
         } else if (S_ISDIR(f_mode)) { //Directory entry
             //Table row for the dir
             printf_s=snprintf(html+pagesize,maxsize,
-                              "<tr style=\"background-color: #DFDFDF;\"><td>d</td><td><a href=\"%s/\">%s/</a></td><td>-</td><td>%s</td></tr>\n",
-                              namelist[i]->d_name, namelist[i]->d_name,last_modified);
+                              "<tr style=\"background-color: %s;\"><td><b>d</b></td><td><a href=\"%s\"><b>%s/</b></a></td><td></td><td>%s</td></tr>\n",
+                              color, namelist[i]->d_name, namelist[i]->d_name,last_modified);
             maxsize-=printf_s;
             pagesize+=printf_s;
+            color = (color == "white" ? "#E0E0E0" : "white");
```

Comment:
If that is important to you, I would recommend setting the color on the directories like #D0D0D0 as before. But then you no longer have alternating light/dark lines if the directories fall on a light row... Instead, I made the font bold, so they stand out that way.

But, I don't mean to push all these things onto you, that's just personal taste. There are 2 things that I think are wrong now:
1. Using XML headers (That was the first commit, and I did a pull request for that one. I didn't mean for you to pull all the other changes...)
2. The time says GMT, which is only correct if the local time zone is GMT. (I live in Thailand...) I didn't know how to send that commit as a separate issue... Sorry, not so handy with git.